INSTALL

```diff
@@ -1893,105 +1265,44 @@ gives it the system app folder icon).  If you have the correct permissions you
 may want to build straight into your /Applications folder. The instructions
 below assume you are building into a ${HOME}/Applications directory.
 
-You have two interactive options for configuring the build: ccmake or run
-Terminal commands. ccmake is a curses interface inside Terminal for CMake and
-allows a tabular layout for viewing and editing ALL available QGIS source CMake
-options. To get started initially run the Terminal method.
-
 In a Terminal cd to the qgis source folder previously downloaded, then:
 
   mkdir build
   cd build
-  cmake -D CMAKE_INSTALL_PREFIX=~/Applications \
-  -D CMAKE_BUILD_TYPE=MINSIZEREL -D ENABLE_TESTS=FALSE \
-  -D SPATIALINDEX_LIBRARY=/usr/local/lib/libspatialindex.dylib \
-  -D SPATIALINDEX_INCLUDE_DIR=/usr/local/include/spatialindex \
-  -D QWT_LIBRARY=/usr/local/qwt-6.0.2/lib/libqwt.dylib \
-  -D QWT_INCLUDE_DIR=/usr/local/qwt-6.0.2/include \
-  -D BISON_EXECUTABLE=/usr/local/bin/bison \
-  ..
+  
+  QGIS_DEPS_VERSION=0.3.0;\
+  QT_VERSION=5.14.1;\
+  PATH=/opt/QGIS/qgis-deps-${QGIS_DEPS_VERSION}/stage/bin:$PATH;\
+  cmake \
+    -DCMAKE_INSTALL_PREFIX=~/Applications \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DQGIS_MAC_DEPS_DIR=/opt/QGIS/qgis-deps-${QGIS_DEPS_VERSION}/stage \
+    -DCMAKE_PREFIX_PATH=/opt/Qt/${QT_VERSION}/clang_64 \
+    ..
 
 Note: Don't forget the .. on the last line, which tells CMake to look for the
 source files in one directory up.
 
+Note: Double check on the screen output that all libraries are picked from QGIS-Deps `/opt/QGIS`
+and not from system `/usr/lib` or Homebrew's `/usr/local/` or system Frameworks `/Library/Frameworks/`.
+Especially check Proj, GDAL, sqlite3 and Python paths.
+
 After the initial Terminal configure, you can use ccmake to make further changes:
 
   cd build
   ccmake ..
```

Comment:
maybe add it to the install list above (but no biggie I guess, if you prefer to keep the list minimal)

Reply:
isn't ccmake part of cmake package?

Reply:
oops, indeed